gef.py

```diff
@@ -8784,6 +8813,59 @@ def do_invoke(self, argv):
 
         return
 
+
+@register_command
+class HighlightCommand(GenericCommand):
+    """Highlight Text."""
+    _cmdline_ = "highlight"
+    _syntax_ = "highlight (list|del|add|clear|regex) [match] [color]"
+    _aliases_ = ["hh", "hl"]
+
+    def clear_highlight_table(self):
+        HIGHLIGHT_TABLE.clear()
+
+    def del_highlight_table_item(self, item):
+        HIGHLIGHT_TABLE.pop(item, None)
+
+    def add_highlight_table_item(self, match, color):
+        HIGHLIGHT_TABLE[match] = color
+
+    def set_highlight_regex(self, enabled):
+        global HIGHLIGHT_REGEX_ENABLED
+        HIGHLIGHT_REGEX_ENABLED = enabled == "on"
+
+    def print_highlight_table(self, raw=False):
+        if raw:
+            pprint.pprint(HIGHLIGHT_TABLE)
+        else:
+            for key, value in HIGHLIGHT_TABLE.items():
+                print('{}'.format(Color.colorify(key, value)))
+
+    def do_invoke(self, argv):
+        if len(argv) == 0:
```

Comment:
consider adding subcommands to make this easier. See `GlibcHeapChunksCommand` vs `GlibcHeapCommand`.